### ISSUE TITLE
how to do complex queries?

### ISSUE THREAD

**Issue by darkyen** [victorquinn/dynasty#82]
Can you redirect me to an example ?

**Comment by mattandrews** [comment]
Yep, this has left me lost too. I want to retrieve all items where the sort key is less than `$foo`. It seems like Dynasty's `find()` methods only do direct equality comparison whereas DynamoDB offers lots of comparison modifiers.

**Comment by jeffhuys** [comment]
It's a shame this hasn't been answered yet. This library seems very promising, but unfortunately we have to decide to not use Dynasty because of the lack of documentation and examples.
